Xamarin.Forms.Core.UnitTests/BindableLayoutTests.cs

```diff
@@ -13,12 +15,12 @@ public class BindableLayoutTests : BaseTestFixture
 		[Test]
 		public void TracksEmpty()
 		{
-			var layout = new StackLayout
+			StackLayout layout = new StackLayout
 			{
```

Comment:
Why this change? First, since `new Type` is being used, not using variable type and using var instead is recommended way by Microsoft guidelines. Second, you should not do this kind of changes.

Reply:
Sorry, those changes were unintentional from VS doing a cleanup. I'll fix.